shared/ortcustomops.cc

```diff
@@ -1,11 +1,32 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include <mutex>
 #include <set>
 
 #include "onnxruntime_extensions.h"
 #include "ocos.h"
 
+struct OrtCustomOpDomainDeleter {
```

Comment:
This code snippet was taken from https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/testdata/custom_op_library/custom_op_library.cc

Reply:
Indeed it's testing code, but it shows how you have to take care of registering and  releasing the custom domains